src/main/java/technology/semi/weaviate/client/v1/data/util/ObjectsPath.java

```diff
@@ -148,10 +148,16 @@ private void addQueryAdditionals(Params params, List<String> pathParams, List<St
     }
   }
 
-  private void addQueryLimit(Params params, List<String> pathParams, List<String> queryParams) {
+  private void addQueryLimitOffsetAfter(Params params, List<String> pathParams, List<String> queryParams) {
```

Comment:
could you please split this method into 3 separate ones?
my intention was to have single method per param

Reply:
sure, will do that